conf/task_defaults/bigearthnet.yaml

```diff
@@ -9,8 +9,8 @@ experiment:
     in_channels: 14
```

Comment:
From what I can tell, `conf/task_defaults/*.yaml` are only used during testing. We should consider moving them to `tests/conf`

Reply:
They are used here -- https://github.com/microsoft/torchgeo/blob/main/train.py#L97. They provide the default arguments for each trainer.

Reply:
See https://github.com/microsoft/torchgeo/pull/45

Reply:
I think it makes sense to re-do the configs -- I'll open an issue describing this